### DISCUSSION TITLE
谁可以给我提供2.3.2版本的副本

### DISCUSSION THREAD

**Discussion by iPodtouch6557** [Meloong-Git/PCL#1457] [提问] [answered]
#666 的问题我仍然没有解决，怎样都不可以，我很难（）
然而可用副本不小心被我删掉了，我只有这唯一的副本，现在没办法用pcl了……
给我个2.3.2的副本，或者详细帮我解决问题，谢谢=(

**Comment by WForst-Breeze** [comment, marked as answer]
我这里正好有 Snapshot 2.3.2 的副本，但是还请您注意，由于版本有些久远，因此我并不建议您使用此版本，已有很多功能无法正常使用。如果您需要，我可以私发给您。

**Reply by WForst-Breeze** [reply]
方便留个电子邮件地址或 QQ 号吗？如果后续有问题可以随时找我 >v<

**Reply by iPodtouch6557** [reply]
> 方便留个电子邮件地址或 QQ 号吗？如果后续有问题可以随时找我 >v<

邮箱吧，liuyuer233@outlook.com,电邮发我就行
先有一个能用的用着，后续再详细解决问题

**Reply by WForst-Breeze** [reply]
发到您邮箱了，您看看收到了吗

**Reply by iPodtouch6557** [reply]
> 发到您邮箱了，您看看收到了吗
收到了，但是又被删除了（）
我不管了，谢谢你的帮助~
